rice-middleware/impl/src/main/java/org/kuali/rice/kim/bo/ui/KimDocumentBoActivatableToFromBase.java

```diff
@@ -15,14 +15,17 @@
  */
 package org.kuali.rice.kim.bo.ui;
 
+import org.kuali.rice.krad.data.jpa.converters.BooleanYNConverter;
+
 import javax.persistence.Column;
+import javax.persistence.Convert;
 import javax.persistence.MappedSuperclass;
 import javax.persistence.Transient;
 import java.sql.Timestamp;
 
 /**
- * This is a description of what this class does - shyu don't forget to fill this in. 
- * 
+ * This is a description of what this class does - shyu don't forget to fill this in.

```

Comment:
😬

Reply:
Yeah, this kind of thing is all over the place.  You may find more success ignoring whitespace changes when reviewing the PR since IntelliJ changed whitespace on a bunch of lines I didn't actually modify while I was working through things.

Reply:
This is what code looks like that *didn't* go through a PR/code review ;)